gateway-ha/src/main/java/com/lyft/data/gateway/ha/clustermonitor/ActiveClusterMonitor.java

```diff
@@ -80,38 +83,47 @@ public void start() {
   private ClusterStats getPrestoClusterStats(ProxyBackendConfiguration backend) {
     ClusterStats clusterStats = new ClusterStats();
     clusterStats.setClusterId(backend.getName());
-
-    String target = backend.getProxyTo() + "/v1/cluster";
-    try {
-      URL url = new URL(target);
-      HttpURLConnection conn = (HttpURLConnection) url.openConnection();
-      conn.setConnectTimeout((int) TimeUnit.SECONDS.toMillis(BACKEND_CONNECT_TIMEOUT_SECONDS));
-      conn.setReadTimeout((int) TimeUnit.SECONDS.toMillis(BACKEND_CONNECT_TIMEOUT_SECONDS));
-      conn.setRequestMethod(HttpMethod.GET);
-      conn.connect();
-      int responseCode = conn.getResponseCode();
-      if (responseCode == HttpStatus.SC_OK) {
-        clusterStats.setHealthy(true);
-        BufferedReader reader =
-            new BufferedReader(new InputStreamReader((InputStream) conn.getContent()));
-        StringBuilder sb = new StringBuilder();
-        String line;
-        while ((line = reader.readLine()) != null) {
-          sb.append(line + "\n");
+    // The V1_NODE_PATH is used in 331 while V1_CLUSTER_PATH is used in 318
+    // TODO: Remove V1_CLUSTER_PATH once we're upgraded all clusters.
```

Comment:
We don't need to remove the path for V1, instead, we can have a mapping between presto version and the API path
both the `/v1/status` and `v1/info` will provide the Presto version

Reply:
I'll just get rid of the old path as we're upgrading everything to 332. No need to muck up the code with version dependent switches.